### ISSUE TITLE
Use https within swagger ui

### ISSUE THREAD

**Issue by mherz** [stoicflame/enunciate#246]
If I configure Swagger within enunciates modules tag, I'm able to customize the used host and basePath. But how am I able to change the used protocol if I want to use https instead of http?

Example:

    <modules>
        <swagger host="myservice.com" basePath="/serviceEndpoint"/>
    </modules>

If I configure swagger like this, swagger ui will use the url 'http://myservice.com/serviceEndpoint'. But I would like to use https instead of http.

**Comment by stoicflame** [comment]
It's not clear to me how to configure Swagger to use https. Can you suggest a way to generate the swagger.json file to allow this?

**Comment by mherz** [comment]
The Swagger JSON file already contains the following:

    "host" : "myservice.com",
    "basePath" : "/serviceEndpoint",

To work with https, I think we would need also:
    "schemes": ["https"],

In case both is allowed, we only need to add "http" to the JSON array. I hope it's comprehensible. Otherwise I will provide a full example file later.

**Comment by stoicflame** [comment]
Fixed at 1bafdc3.

```xml
<enunciate>
  <modules>
    <swagger>
      <scheme>https</scheme>
      <scheme>http</scheme>
```

**Comment by stoicflame** [comment]
[Enunciate 2.2 is now available](https://github.com/stoicflame/enunciate/releases/tag/v2.2.0).
